Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Keep golang version aligned with latest yocto release
-FROM golang:1.17.13-bullseye as builder
+FROM golang:1.22.2-bullseye as builder
```

Comment:
Please keep the version aligned to yocto, see comment above.

Coincidentally, we are discussing this in the [upcoming client team meeting](https://docs.google.com/document/d/1neXnS16brsb7Jdn-Sg-f1Wyoyo_O5MJIAW7_FzTKpfY/edit#heading=h.ih2nzgp6q407).

Reply:
Please keep in mind that `go1.17` has been out of support for almost 2 years. Is it not possible to use the most recent version of Go when compiling Yocto?

Reply:
Yocto maintains the toolchain, porting security fixes (same as for example Debian or other OSs do for the same time period that they support the OS).

Fixing the version here does have any impact in the final yocto build nor in the deb packages build.

Fixing the version here is only a way for us (me) to early catch when updates will break in today *latest* yocto LTS. Then we can decide to accept the breakage and patch the code there or reject the update for the time being. It is a difficult balance - to give you a small nitpick: we still support `dunfell` LTS which uses golang 1.14 and it is really hard and time consuming delivering our software there.